### PR TITLE
fixed ldap cron job in Installation manual

### PR DESCRIPTION
**Pull request by voroyam**
there was a wrong cron job for user:sync. 

I also added a log file for the output of user:sync

changed header levels for better overview

removed php reference in Cron Jobs. You don't need php for system cron and user sync.

### CODE REVIEW

File: modules/admin_manual/pages/installation/ubuntu_18_04.adoc [hunk 1]
```diff
@@ -148,37 +147,40 @@ occ maintenance:install \
...
+echo "*/15  *  *  *  * /var/www/owncloud/occ system:cron" \
   > /var/spool/cron/crontabs/{webserver-user}
 chown {webserver-user}.crontab /var/spool/cron/crontabs/{webserver-user}
 chmod 0600 /var/spool/cron/crontabs/{webserver-user}
 ----
 
 [NOTE]
 ====
-If you need to sync your users from an LDAP or Active Directory Server, add this additional xref:configuration/server/background_jobs_configuration.adoc[Cron job].
+If you need to sync your users from an LDAP or Active Directory Server, add this additional xref:configuration/server/background_jobs_configuration.adoc[Cron job]. Every 15 minutes this cron job will sync LDAP users in ownCloud and disable the ones who are not available for ownCloud. Additionally you get a log file in `var/log/ldap-sync/user-sync.log` for debugging.
```
Comment (ho4ho) [inline modules/admin_manual/pages/installation/ubuntu_18_04.adoc:170]:
```suggestion
If you need to sync your users from an LDAP or Active Directory Server, add this additional xref:configuration/server/background_jobs_configuration.adoc[Cron job]. Every 15 minutes this cron job will sync LDAP users in ownCloud and disable the ones who are not available for ownCloud. Additionally you get a log file in `/var/log/ldap-sync/user-sync.log` for debugging.
```

Comment (mmattel) [reply modules/admin_manual/pages/installation/ubuntu_18_04.adoc]: good catch 👍

**Comment by voroyam** [comment]
to all of them since system:cron was introduced.

I look up where it was

**Comment by voroyam** [comment]
10.3.0

**Comment by phil-davis** [comment]
Backports:
10.5 #2733 
10.4 #2734

**Pull request merged**